package.json

```
@@ -15,15 +15,16 @@
   "license": "MIT",
   "devDependencies": {
     "babel": "^6.23.0",
```

```suggestion
    "babel": "^6.23.0",
```

Don't think babel is used in this